token/transfer-hook/cli/src/meta.rs

```diff
@@ -0,0 +1,318 @@
+use {
+    serde::{Deserialize, Serialize},
+    solana_sdk::pubkey::Pubkey,
+    spl_tlv_account_resolution::{account::ExtraAccountMeta, seeds::Seed},
+    std::{fmt, path::Path, str::FromStr},
+    strum_macros::{EnumString, IntoStaticStr},
+};
+
+#[derive(Debug, Clone, Copy, PartialEq, EnumString, IntoStaticStr, Serialize, Deserialize)]
+#[serde(rename_all = "kebab-case")]
+#[strum(serialize_all = "kebab-case")]
```

Comment:
how about we change this to camel case to be consistent with the rest? Right now, we've got snake-case for YAML, camel-case for JSON, and kebab-case for just this enum. That might be pretty confusing to people, so how about just using one standard for the whole thing?

Reply:
I left this in for now, but I can change it. I figured since it was a string literal it was OK and made sense, especially because it's the same across JSON and YAML, and the CLI arg.

But I can change it to allow for both camel and snake instead.

Reply:
I would eliminate all snake-case support if I were you. Maybe this can keep kebab-case for consistency with other command-line tools that we provide, but camel-case should be the priority. From what I see, YAML doesn't have hard conventions for casing, so let's use that to our advantage to avoid multiple formats

Reply:
Weird I swore YAML was always snake case. But you're right, looks like it's not strict. I agree let's get rid of snake case for sure.